### PR TITLE
Add exolix

### PR DESCRIPTION
**Pull request by devexolix**
## Description

-

## How Has This Been Tested?

-

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by IAmKio** [comment]
@devexolix sorry this looks like it's rendering an `iframe` which is not allowed:
https://github.com/pillarwallet/x/pull/173/files#diff-5a58296effc18582ddc8fca3b692f072fbed94a29ab969790e6845fc4dae82dfR93

**Comment by devexolix** [comment]
> @devexolix sorry this looks like it's rendering an `iframe` which is not allowed: https://github.com/pillarwallet/x/pull/173/files#diff-5a58296effc18582ddc8fca3b692f072fbed94a29ab969790e6845fc4dae82dfR93

Yes, we use iframe, but it links to your repository in folder "/public/exolix-static" and you have all the source code and we haven't plugged in any outside code.

File: src/apps/index.ts [hunk 1]
```diff
@@ -31,6 +31,8 @@ export const loadApp = (appId: string) => {
 export const loadApps = (allowedApps: string[]) => {
   const loadedApps: Record<string, AppManifest> = {};
 
+  console.log(allowedApps);
+
```
Comment (IAmKio) [inline src/apps/index.ts:35]: Could we remove this please

File: src/apps/exolix/index.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '100vh',
+      }}
+    >
+      <iframe srcDoc={templ} width="100%" style={{ height: '100%' }}></iframe>
+    </div>
```
Comment (IAmKio) [inline src/apps/exolix/index.tsx:94]: Is it not possible for you to export `templ` as a React Component module and then return that here? Is there a reason it needs to be in an `iframe`?

**Comment by devexolix** [comment]
@IAmKio Yes, we pack build our site and add to app, and this is why we need use the iframe because the build version need render in HTML tag and have the own DOM.

**Pull request merged**